tests/test_main.py

```diff
@@ -93,6 +93,97 @@ def test_lint_submodule_included_fail(submodule_repository, stringio):
     assert ":-(" in stringio.getvalue()
 
 
+def test_lint_meson_subprojects(fake_repository, stringio):
+    """Run a successful lint."""
+    (fake_repository / "meson.build").write_text(
+        "SPDX-FileCopyrightText: 2022 Jane Doe\n"
```

Comment:
This probably throws a number of lint errors. Please escape it in some way, e.g. by splitting the string or making it lowercase (and uppercase later). There are a number of examples in other tests.

Also applies to lines below, e.g. 108/109.

Reply:
Sorry, I've no idea what the problem is, could you be more specific?

    pytest -k meson

runs through without errors...

Reply:
Yes, but running `reuse lint` over the project fails. This is also shown in the Actions (also `black` seems to demand changes).

You could escape it by this:
https://github.com/fsfe/reuse-tool/blob/7ac2cd03b18a45463ae353763c7ef43757f510b6/tests/test_project.py#L106-L108

Reply:
Fixed in [78e172d](https://github.com/fsfe/reuse-tool/pull/496/commits/78e172d02674745edeb1212b6ff904ff3a8750de)